src/compas/scene/scene.py

```diff
@@ -43,38 +43,36 @@ class Scene(Tree):
     @property
     def __data__(self):
         # type: () -> dict
-        items = {str(object.item.guid): object.item for object in self.objects if object.item is not None}
         return {
             "name": self.name,
             "root": self.root.__data__,  # type: ignore
-            "items": list(items.values()),
+            "datastore": self.datastore,
         }
 
     @classmethod
     def __from_data__(cls, data):
         # type: (dict) -> Scene
-        scene = cls(data["name"])
-        items = {str(item.guid): item for item in data["items"]}
-
-        def add(node, parent, items):
+        scene = cls(data["name"], datastore=data["datastore"])
```

Comment:
in this version, we still allow direct serialization and deserialization of `datastore` attribute